### DISCUSSION TITLE
script to empty a font from it's glyphs paths

### DISCUSSION THREAD

**Discussion by edi0th** [fontforge/fontforge#4742] [Q&A] [answered]
Hi ! It's been a few days since I try to acheive this thanks to doc, but it's impossible for me to find the answer. I would like to make a script to open a otf, and remove all glyphs paths but without removing each glyphs width, spacing and kerning. 

Does someone knows how to do that ?
If I use clear(), then everything is erased...

Thanks in advance !
E.

**Comment by ctrlcctrlv** [comment, marked as answer]
What about…

```python3
import fontforge as ff
f = ff.activeFont()

for g in f.glyphs():
    g.layers["Fore"] = ff.layer()
```

**Reply by edi0th** [reply]
That was...Simple :smile: Thanks a lot ! I didn't new you could reset a layer like this ! Thanks
